_posts/2020-05-05-churn-vs-complexity-vs-coverage.markdown

```diff
@@ -0,0 +1,252 @@
+| lib/skunk/cli/commands/default.rb                   | 166.44                     | 1.6643999999999999         | 3                          | 0.5548                     | 0                          |
+| lib/skunk/cli/application.rb                        | 139.2                      | 1.392                      | 3                          | 0.46399999999999997        | 0                          |
+| lib/skunk/cli/command_factory.rb                    | 97.6                       | 0.976                      | 2                          | 0.488                      | 0                          |
+| test/test_helper.rb                                 | 75.2                       | 0.752                      | 2                          | 0.376                      | 0                          |
+| lib/skunk/rubycritic/analysed_module.rb             | 48.12                      | 1.7184                     | 2                          | 0.8592                     | 72.72727272727273          |
+| test/lib/skunk/cli/commands/status_reporter_test.rb | 45.6                       | 0.456                      | 1                          | 0.456                      | 0                          |
+| lib/skunk/cli/commands/base.rb                      | 29.52                      | 0.2952                     | 3                          | 0.0984                     | 0                          |
+| lib/skunk/cli/commands/status_reporter.rb           | 8.0                        | 7.9956                     | 3                          | 2.6652                     | 100.0                      |
+| test/lib/skunk/rubycritic/analysed_module_test.rb   | 2.63                       | 2.6312                     | 2                          | 1.3156                     | 100.0                      |
+| lib/skunk.rb                                        | 0.0                        | 0.0                        | 2                          | 0.0                        | 0                          |
+| lib/skunk/cli/options.rb                            | 0.0                        | 0.0                        | 2                          | 0.0                        | 0                          |
+| lib/skunk/version.rb                                | 0.0                        | 0.0                        | 2                          | 0.0                        | 0                          |
+| lib/skunk/cli/commands/help.rb                      | 0.0                        | 0.0                        | 2                          | 0.0                        | 0                          |
++-----------------------------------------------------+----------------------------+----------------------------+----------------------------+----------------------------+----------------------------+
+
+```
+
```

Comment:
Maybe you can add some hint here about how you can think about that score. I mean, looking at that it's not clear if 166.44 is only the worst score for this project (but a good score anyway) or if it's a bad score in general (I know I heard you talk about like 10.000 skunk score or something like that on your talks).

Reply:
@arielj I don't think that's relevant to the point I'm trying to make. Also, I don't have enough data to give context about that. 

I tried to expand with an example. Let me know if that helps.

Reply:
Sure, I think the example is good enough, I was just wondering because I heard you talked about those 10000 or 1000 skunk score some time (not really sure which one haha) and I was thinking maybe with the experience you had using it you saw some extreme values. But I think it's good right now 👍